omod/src/main/webapp/pages/visit/visit.gsp

```diff
@@ -37,8 +37,8 @@
     ui.includeJavascript("pihcore", "visit/vaccinations.js")
     ui.includeJavascript("pihcore", "visit/visit.js")
     ui.includeJavascript("pihcore", "visit/encounterTransaction.js")
-    ui.includeJavascript("pihcore", "visit/encounterTypeConfig.js")
     ui.includeJavascript("pihcore", "services/configService.js")
+    ui.includeJavascript("file", "configuration/pih/scripts/visit/encounterTypeConfig.js", null, true)
```

Comment:
See https://issues.openmrs.org/browse/UIFR-214 for the explanation behind "null" and "true"

Reply:
In keeping with my suggestions above, I'd inline that information into comments.